tidb-cloud/terraform-migrate-cluster-resource.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Migrate Cluster Resource to Serverless/Dedicated Cluster Resource
+summary: Learn how to migrate cluster resource to serverless/dedicated cluster resource.
+---
+
+# Migrate Cluster Resource to Serverless/Dedicated Cluster Resource
+
+You will learn how to migrate cluster resource to serverless/dedicated cluster resource in this document.
+
+## Prerequisites
+
+- [Update TiDB Cloud Terraform Provider at least x.x.x]()
+
+## Step 1. Select the `tidbcloud_cluster` resource to migrate
+
+Use the command below to get all resources.
+
+```shell
+terraform state list | grep "tidbcloud_cluster"
+```
+
+Select the cluster resource to migrate and execute the command below to get the `id` for later use.
+
+```shell
+terraform state show tidbcloud_cluster.serverless_tier_cluster | grep ' id '
```

Comment:
The doc below use `${your_target_cluster_resource}`, here we can also use `${your_target_cluster_resource}` to keep the consistency of the doc